Kratos identity Management

valentinyanakiev opened the pull request:
```gql 
query {
  platformAdmin {
    identity {
      identities(filter: ALL) {
        id
        email
        firstName
        lastName
        createdAt
        verificationStatus
        isV
      }
    }
  }
}
```

```gql
mutation {
  adminIdentityDeleteKratosIdentity(kratosIdentityId:"8298b1ab-018d-434d-9205-7dc4f4a104ce")
}
```

```bash
npm run start:services
```

then visit http://localhost:8188/identities

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
  - Platform Admin: manage identities — browse with verification filters, list unverified identities, view identity details, and delete identities via new queries, fields, and mutations.
  - New identity DTOs and services to expose identity details (id, email, name, createdAt, verification status).

- **Chores**
  - Quickstart/Traefik: added Kratos Admin UI service and exposed port 8188 for access.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

valentinyanakiev committed:
Adds admin identity management module

Introduces a new module for managing admin identities, including retrieving unverified identities from Kratos.

This module enhances platform administration capabilities by providing functionalities to list and manage user identities, specifically focusing on unverified accounts.

valentinyanakiev committed:
Adds identity management features

Adds identity management capabilities to the platform admin interface.
This includes the ability to view and filter identities based on
verification status. It introduces new resolvers, services, and DTOs
to facilitate querying and displaying identity information.

valentinyanakiev committed:
Adds Kratos admin UI and delete identity

This commit introduces the Kratos admin UI for identity management.

It also implements the ability to delete Kratos identities by email
address. The changes include adding a new endpoint for deleting identities
and updating the Kratos service to include the delete functionality.
The matrix adapter image was also updated.

valentinyanakiev committed:
Refactors identity management for Kratos

Updates identity management to directly utilize Kratos identities.
This removes the abstraction of PlatformIdentityDto and streamlines
operations by using KratosIdentityDto throughout the platform admin
module. It also changes the delete identity functionality to use the
Kratos ID instead of email.

valentinyanakiev committed:
Update src/platform-admin/core/identity/admin.identity.resolver.mutations.ts

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

techsmyth commented on quickstart-services.yml:
```diff
@@ -243,7 +255,7 @@ services:
     container_name: alkemio_dev_matrix_adapter
     hostname: matrix-adapter
-    image: alkemio/matrix-adapter:v0.6.1
+    image: vyanakiev/matrix-adapter-circuit:v0.0.1
```
why do we have an image from your container registry?

techsmyth commented on quickstart-services.yml:
```diff
@@ -92,6 +93,17 @@ services:
         target: /etc/config/kratos
     command: serve -c /etc/config/kratos/kratos.yml --dev --watch-courier
 
+  kratos-admin-ui:
```
To be also added into prod or not?

techsmyth reviewed (dismissed):
Working nicely! One comment that looks like a mistake, one question. Can be merged...

valentinyanakiev committed:
Updates matrix-adapter image tag

Updates the matrix-adapter image tag to the latest version.
This ensures the quickstart environment uses the most recent
features and bug fixes.

valentinyanakiev committed:
Merge branch 'develop' into kratos-identity
